hnn_core/batch_simulate.py

```diff
@@ -19,7 +19,10 @@ def __init__(self, set_params, net_name='jones', tstop=170,
                  dt=0.025, n_trials=1, record_vsec=False,
                  record_isec=False, postproc=False, save_outputs=False,
                  save_folder='./sim_results', batch_size=100,
-                 overwrite=True, summary_func=None):
+                 overwrite=True, summary_func=None,
+                 save_dpl=True, save_spiking=False,
+                 save_LFP=False, save_voltages=False,
```

Comment:
```suggestion
                 save_lfp=False, save_voltages=False,
```
Just to keep consistent as we generally reserve uppercase characters for classes